nested_K8sCustomIngressTlsHostsHaveDefinedDomainSuffix.bicep

```diff
@@ -84,57 +84,6 @@ resource pdK8sCustomIngressTlsHostsHaveDefinedDomainSuffix 'Microsoft.Authorizat
         }
         defaultValue: {
         }
-        schema: {
```

Comment:
have we considered aligning the schema to [this example](https://learn.microsoft.com/en-us/azure/governance/policy/concepts/definition-structure#example-2)?

Reply:
Thanks, I will investigate.  The deployment had been giving a warning about the "schema" key.

Reply:
(retaining for reference)
![image](https://github.com/mspnp/aks-baseline/assets/82831332/1a22f1ad-5b2f-4c8d-89ba-6c73ed37a824)

Reply:
Going to remove this file to address later